plugins/cisco_umbrella_destinations/icon_cisco_umbrella_destinations/util/api.py

```diff
@@ -1,11 +1,12 @@
 import json
-from json import JSONDecodeError
 from logging import Logger
 from typing import Optional
 import requests
 from requests.auth import HTTPBasicAuth
 from insightconnect_plugin_runtime.exceptions import PluginException
 
+ERROR_MSG = "Invalid input data, ensure the information you are inputting is correct"
```

Comment:
```suggestion
ERROR_MSG = "Invalid input data, ensure the input is correct"
```
It may not be user-input data here, so this should be adjusted